focus-ios/focus-ios-tests/XCUITest/URLValidationTest.swift

```diff
@@ -0,0 +1,31 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import XCTest
+
+class URLValidationTest: BaseTestCase {
+    override func setUp() {
+        super.setUp()
+        continueAfterFailure = true
+    }
+
+    let urlTypes = ["mozilla.org", "mozilla.org/", "https://mozilla.org", "mozilla.org/en", "mozilla.org/en-",
```

Comment:
can we add an url with `http://foo.bar` and `http://foo.bar/`

Reply:
Both cases are covered with: "https://mozilla.org" and "https://mozilla.org/"

Reply:
those have `https`, I meant just `http`

Reply:
i will add it

Reply:
thanks!

Reply:
updated